packages/gestalt/dist/index.d.ts

```diff
@@ -1411,7 +1413,7 @@ interface PopoverProps {
   children?: Node | undefined;
   color?: 'deprecatedBlue' | 'red' | 'white' | 'darkGray' | undefined;
   id?: string | undefined;
-  idealDirection?: FourDirections | undefined;
+  idealDirection?: FiveDirections | undefined;
```

Comment:
lol, can we just call this PopoverDirections?

Reply:
DONE